tests/test_vis.py

```diff
@@ -458,6 +458,7 @@ def test_vegalite_default_actions_registered_2(global_var):
 
 
 def test_matplotlib_default_actions_registered(global_var):
+    df = pytest.car_df
     lux.config.plotting_backend = "matplotlib"
```

Comment:
why do we need 461 and 463? what changed?

Reply:
Yeah that is redundant, will remove one of the unnecessary calls here.